apps/condo/domains/miniapp/schema/preprocessors/B2BAppAccess.js

```diff
@@ -0,0 +1,505 @@
+    return `{ ${pathToOrganizationId.join(' { ')}${' }'.repeat(pathToOrganizationId.length)}`
+}
+
+/**
+ *
+ * @param listKey
+ * @param pathToOrganizationId
+ * @return {string}
+ *
+ * @example
+ * Input: ('Ticket', ['organization', 'id'])
+ * Output: `
+ *         query getAllTickets ($where: TicketWhereInput, $first: Int = 1) {
+ *             objs: allTickets(where: $where, first: $first) { organization { id } }
+ *         }
+ *     `
+ */
+const generateGqlQueryAsString = (listKey, pathToOrganizationId) => {
+    if (!isString(listKey) || listKey.trim().length < 1) throw new Error('"listKey" must not be empty string!')
+    if (!pathToOrganizationId || !isArray(pathToOrganizationId) || isEmpty(pathToOrganizationId)) throw new Error('"pathToOrganizationId" should not be empty array')
+
+    return `
+        query getAll${pluralize.plural(listKey)} ($where: ${listKey}WhereInput, $first: Int = 1) {
+            objs: all${pluralize.plural(listKey)}(where: $where, first: $first) ${generateGqlDataPart(pathToOrganizationId)}
+        }
+    `
+}
+
+const ALL_GENERATED_GQL_QUERIES = new Map()
+
+/**
+ *
+ * @param pathToOrganizationId {string[]}
+ * @return {*|DocumentNode}
+ *
+ * @example
+ * Input: ['ticket', 'organization', 'id']
+ * Output: gql`
+ *         query getAllTickets ($where: TicketWhereInput, $first: Int = 1) {
+ *             objs: allTickets(where: $where, first: $first) { organization { id } }
+ *         }
+ *     `
+ */
+const generateGqlQuery = (pathToOrganizationId) => {
+    if (!pathToOrganizationId || !isArray(pathToOrganizationId) || isEmpty(pathToOrganizationId)) throw new Error('"pathToOrganizationId" should not be empty array')
+    if (pathToOrganizationId.length < 2) throw new Error(`To generate gql "pathToOrganizationId" must contain at least two elements! But was ${pathToOrganizationId}`)
+
+    const gqlKey = pathToOrganizationId.join(':')
+
+    // get already generated gql query to reuse them
+    if (ALL_GENERATED_GQL_QUERIES.has(gqlKey)) {
+        return ALL_GENERATED_GQL_QUERIES.get(gqlKey)
+    }
+
+    const listKey = pathToOrganizationId[0].charAt(0).toUpperCase() + pathToOrganizationId[0].slice(1)
+
+    const generatedGql = gql`${generateGqlQueryAsString(listKey, pathToOrganizationId.slice(1))}`
+
+    // save generated gql query to reuse them
+    ALL_GENERATED_GQL_QUERIES.set(gqlKey, generatedGql)
+
+    return generatedGql
+}
+
+const canManageByServiceUser = async ({ authentication: { item: user }, listKey, originalInput, itemId, operation, context }, schemaConfig) => {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (!listKey) return false
+
+    const isBulkRequest = Array.isArray(originalInput)
+
+
+    if (isBulkRequest) return false
+
+    const pathToOrganizationId = get(schemaConfig, 'pathToOrganizationId', ['organization', 'id'])
+    if (!isArray(pathToOrganizationId) || isEmpty(pathToOrganizationId)) return false
```

Comment:
`pathToOrganizationId` cannot be empty because you set default value in `get`

Reply:
if you can set `pathToOrganizationId: []`